src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] [REQUIRED] Kindly refactor the value of your Action Type to use the [ducks pattern](https://github.com/erikras/ducks-modular-redux):
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`